src/TimePicker.elm

```diff
@@ -243,6 +262,40 @@ update settings msg (TimePicker ({ value } as model)) =
             ( TimePicker { model | inputText = Nothing, value = updatedValue }, timeEvent )
 
 
+defaultHour : Settings -> Int
+defaultHour settings =
+    Maybe.withDefault 0 (find (settings.isHourDisabled >> not) allHours)
+
+
+defaultMinute : Settings -> Int
+defaultMinute settings =
+    Maybe.withDefault 0 (find (settings.isMinuteDisabled >> not) allMinutes)
+
+
+defaultSecond : Settings -> Int
+defaultSecond settings =
+    Maybe.withDefault 0 (find (settings.isSecondDisabled >> not) allSeconds)
+
+
+{-| Find the first element that satisfies a predicate and return
+Just that element. If none match, return Nothing.
+find (\\num -> num > 5) [2, 4, 6, 8] == Just 6
+<https://github.com/circuithub/elm-list-extra/blob/3.7.1/src/List/Extra.elm>
```

Comment:
I would reference the more updated: https://github.com/elm-community/list-extra/tree/8.2.2

From what I can tell, you should pull out the attribution from the LICENSE file (there are two names to add from there) and just call them out as being the original authors of this. Since that project and mine are MIT the rest should be okay.

Reply:
I didn't notice actual contributors in the LICENSE file. There are around ~50 contributors to list-extra, so I just linked the github repo in the comment. Let me know if you don't think this is enough.

Reply:
Just copy this line in: `Copyright (c) 2016 CircuitHub Inc., Elm Community members` and use this url instead: https://github.com/elm-community/list-extra/blob/8.2.2/src/List/Extra.elm

Reply:
done.